examples/user_interfaces/fourier_demo_wx.py

```diff
@@ -146,7 +168,9 @@ def mouseDown(self, evt):
             self.state = 'time'
         else:
             self.state = ''
-        self.mouseInfo = (evt.xdata, evt.ydata, max(self.f0.value, .1), self.A.value)
+        self.mouseInfo = (
+            evt.xdata, evt.ydata, max(
+                self.f0.value, .1), self.A.value)
```

Comment:
This isn't really readable. I would put the first two items onto the first line, then the max() on the second line and the last item on the third line, and line it all up with the opening parens.